config/config.py

```diff
@@ -117,4 +115,11 @@ def __getitem__(self, item):
     def __setitem__(self, key, value):
         if self._config_was_set:
             raise RuntimeError('You can\'t change the configuration after it has been loaded from a file.')
-        self._config[key] = value
\ No newline at end of file
+        self._config[key] = value
+
+    def load_parameters(self, cli_args):
```

Comment:
we could use the update method from the dict for each section. This avoid the hardcoding of the key.

self._config['ENGINE'].update(engine_args)
self._config['GAME].update(game_args)